ppa/archive/tests/test_admin.py

```diff
@@ -16,13 +18,13 @@
 }
 
 
-@override_settings(SOLR_CONNECTIONS=TEST_SOLR_CONNECTIONS)
-@patch('ppa.archive.solr.get_solr_connection')
 class TestDigitizedWorkAdmin(TestCase):
 
     def setUp(self):
         self.factory = RequestFactory()
 
+    @override_settings(SOLR_CONNECTIONS=TEST_SOLR_CONNECTIONS)
+    @patch('ppa.archive.solr.get_solr_connection')
```

Comment:
I was using a Django form in case the admin method did anything behind the scenes I thought I might miss with the form, but realized after you pointed it out that a simple `Mock()` would do just fine to supply the required `form` parameter to the admin action. Revised accordingly.